### PR TITLE
Update spanish translation.

### PR DESCRIPTION
**Pull request by resadent**
Added some missing strings.

### CODE REVIEW

File: locales/es_ES/pcsx2_Main.po [hunk 1]
```diff
@@ -1916,8 +1923,15 @@ msgid ""
...
+"Por favor ten en cuenta que las características grabación de entradas de "
+"PCSX2 está todavía incompleta.\n"
+"Como tal, puede haber bugs, problemas de rendimiento e inestabilidad con "
+"ciertos juegos.\n"
+"\n"
+"Esas herramientas son dadas así y deben ser habilitadas a tu propia "
+"discreción."
 
-#: pcsx2/gui/MainMenuClicks.cpp:589
+# :pcsx2/gui/MainMenuClicks.cpp:589
```
Comment (lightningterror) [inline locales/es_ES/pcsx2_Main.po]: Change not necessary.

Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: Typo, Ill address that.

Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: Should be fine now.

File: locales/es_ES/pcsx2_Main.po [hunk 2]
```diff
@@ -8,8 +8,8 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.2.1\n"
```
Comment (lightningterror) [inline locales/es_ES/pcsx2_Main.po]: We can also update the project version to 1.5.0.

Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: Done 👍

**Comment by lightningterror** [comment]
Vsync text option could be translated too

**Comment by lightningterror** [comment]
We were missing a translatable string on master so we made a small update, can the pr be rebased to latest master and the string translated as well?

**Comment by lightningterror** [comment]
Rebase wasn't done properly before pushing new changes, see
https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request
for info regarding how a rebase can be done, if you have trouble let us know here or on discord so we can help out.

**Comment by resadent** [comment]
Alright give me some time, I'm not that good with github now.

**Comment by tadanokojin** [comment]
I hear @turtleli is offering free rebases if you want to go that route ;)

**Comment by resadent** [comment]
Alright it was a little trouble for me to rebase my branch, but I managed to do it. Glad I learned how to :) thanks for the assistance.

> Vsync text option could be translated, EE Cycle skipping too.
I think these are fine now. Can you check? I don't know for sure what lines you were referring to.

**Comment by lightningterror** [comment]
Can you also update the mo files? You can update them with Poedit.

**Comment by resadent** [comment]
> Can you also update the mo files? You can update them with Poedit.

I only have to save the file with poedit right? Do think its done now.

**Comment by lightningterror** [comment]
mo files should be put in bin/Langs/

**Comment by resadent** [comment]
Done.

File: locales/es_ES/pcsx2_Main.po [hunk 3]
```diff
@@ -2069,67 +1949,49 @@ msgid "Recompiler"
...
-msgstr ""
-"Intérprete de la Unidad Vectorial. Lento y no muy compatible. Solo para "
-"diagnósticos."
+msgid "Vector Unit Interpreter. Slow and not very compatible. Only use for diagnostics."
+msgstr "Intérprete de la Unidad Vectorial. Lento y no muy compatible. Solo para diagnósticos."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:188
 msgid "microVU Recompiler"
-msgstr "Recompilador microVU"
+msgstr "recompilador microVU"
```
Comment (lightningterror) [inline locales/es_ES/pcsx2_Main.po]: Small nitpick, Recompilador with capital letter looked better imo.

File: locales/es_ES/pcsx2_Main.po [hunk 4]
```diff
@@ -2069,67 +1949,49 @@ msgid "Recompiler"
...
-msgstr ""
-"Nuevo recompilador de la Unidad Vectorial con compatibilidad muy mejorada. "
-"Recomendado."
+msgid "New Vector Unit recompiler with much improved compatibility. Recommended."
+msgstr "Nuevo recompilador de la Unidad Vectorial con compatibilidad muy mejorada. Recomendado."
 
 #: pcsx2/gui/Panels/CpuPanel.cpp:192
 msgid "superVU Recompiler [legacy]"
-msgstr "Recompilador superVU (legado)"
+msgstr "recompilador superVU [legado]"
```
Comment (lightningterror) [inline locales/es_ES/pcsx2_Main.po]: Here as well.

Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: I changed these to better reflect the behaviour of the English version, but I can revert them.

Comment (lightningterror) [reply locales/es_ES/pcsx2_Main.po]: `Recompilador superVU [legado]` would be good :)

Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: Done. I changed some more strings from capitalized to uncapitalized for the same reason, you can have a look at it if you wish and I can also revert them if you feel its necessary :)

File: locales/es_ES/pcsx2_Main.po (hunk 3 above)
Comment (resadent) [reply locales/es_ES/pcsx2_Main.po]: Fixed.

**Comment by lightningterror** [comment]
I asked @RafaelTrepaUnCarballo to take a look at the pr since he's also a Spanish translator, aside from that it looks good, good job.

**Comment by resadent** [comment]
> I asked @RafaelTrepaUnCarballo to take a look at the pr since he's also a Spanish translator, aside from that it looks good, good job.
> Maybe you two can work together for the translations :)

Thank you. I hope he likes my work :)

File: locales/es_ES/pcsx2_Iconized.po [hunk 5]
```diff
@@ -607,32 +633,43 @@ msgid ""
...
+"rendimiento en juegos poco exigentes, empeora en otros. Las cinemáticas y el "
+"audio pueden entrecortarse o ser saltados."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:41
 #, c-format
 msgid ""
 "75% cyclerate. Slight reduction of CPU requirements. Speedup for less "
 "demanding games, slows down others."
 msgstr ""
+"75% ciclos (tasa). Reducción pequeña de requisitos de CPU. Mejora "
```
Comment (resadent) [inline locales/es_ES/pcsx2_Iconized.po]: That phrase could be expressed in a more understandable way, but I thought the 'c' must come after the % or so poedit told me. Is that the case? If it is not I can change it.

**Comment by RafaelTrepaUnCarballo** [comment]
> > I asked @RafaelTrepaUnCarballo to take a look at the pr since he's also a Spanish translator, aside from that it looks good, good job.
> > Maybe you two can work together for the translations :)
> 
> Thank you. I hope he likes my work :)

I do, you translated everything and before my pr. The only thing I can do for help is revising the translation, looking for typos.

I have some suggestions and questions about the translations:

- Should we translate hack as "arreglo" or "hack"? (a well known anglicism) In PCSX2 you can read both translations in different points.

-Regarding the word "cyclerate", which literally means "tasa de ciclos"... could it be translated as the more known "frecuencia reloj" or that's something entirely different? Personally, I don't like using "ciclos (tasa)".

File: locales/es_ES/pcsx2_Iconized.po (hunk 5 above)
Comment (RafaelTrepaUnCarballo) [reply locales/es_ES/pcsx2_Iconized.po]: En Discord me dijeron que podía ignorar los avisos de poedit relacionados con el "%" y "c", te sugiero lo mismo.

Yo lo redactaría de una forma mas natural, ago como...
_75% de la tasa de ciclos. Reduce un poco los requisitos del procesador. Mejora el rendimiento de juegos poco exigentes, ralentiza los demás._

Hay que destacar que hay una progresión en la descripción de los hacks, los porcentajes más bajos solo se recomiendan en juegos MUY, MUY ligeros, los más altos (<100%) para juegos menos ligeros. La traducción debería reflejar ese cambio con términos como estos:

50% **very** lightweight games --> (juegos muy ligeros)
60% lightweight games --> (juegos ligeros)
75% less demanding games --> (_¿juegos poco exigentes?_) o algo así

File: locales/es_ES/pcsx2_Iconized.po [hunk 6]
```diff
@@ -649,29 +689,41 @@ msgid ""
...
+"compatible."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:87
 msgid ""
 "Mild EE Cycle Skipping. Mild slow down for most games, but may help some "
 "games with mild VU starvation problems run at full speed."
 msgstr ""
+"Ligero salto de ciclos de EE. Pequeña ralentización para la mayoría de "
+"juegos, pero puede ayudar para algunos juegos con problemas de VU "
+"insuficiente a funcionar a velocidad total."
```
Comment (RafaelTrepaUnCarballo) [inline locales/es_ES/pcsx2_Iconized.po]: "puede ayudar para algunos juegos con problemas de VU insuficiente a funcionar a velocidad total"

- ¿"Ayudar para"? ¿No es "Ayudar a"?
- El término informático Starvation normalmente se traduce como "[inanición](https://es.wikipedia.org/wiki/Inanici%C3%B3n_(inform%C3%A1tica))", aunque "VU insuficiente" trasmite la idea

File: locales/es_ES/pcsx2_Iconized.po [hunk 7]
```diff
@@ -649,29 +689,41 @@ msgid ""
...
+"ayudar a algunos otros."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:98
 msgid ""
 "Maximum EE Cycle Skipping. Mostly harmful. May help games with significant "
 "VU starvation problems run at full speed."
 msgstr ""
+"Máximo salto de ciclos de EE. Mayormente perjudicial. Puede mejorar el "
+"rendimiento en los juegos con falta de rendimiento de la unidad vectorial "
+"significativa para que funcionen a velocidad completa."
```
Comment (RafaelTrepaUnCarballo) [inline locales/es_ES/pcsx2_Iconized.po]: "Puede mejorar el rendimiento en los juegos con falta de rendimiento de la unidad vectorial significativa para que funcionen a velocidad completa."

Yo acortaría la frase así:
"Puede ayudar a juegos con problemas significativos de inanición de VU a ejecutarse sin ralentizaciones." (o velocidad completa)

Inanición es un término informático válido y si la frase en inglés solo menciona VU con sus siglas, es mejor que nosotros también las usemos.

File: locales/es_ES/pcsx2_Iconized.po [hunk 8]
```diff
@@ -690,6 +742,11 @@ msgid ""
 "CPU requirements substantially. Lower values will reduce the CPU load "
 "allowing lightweight games to run full speed on weaker CPUs."
 msgstr ""
+"Modifica la frecuencia de la CPUEmotion Engine emulada. Valores más altos "
```
Comment (RafaelTrepaUnCarballo) [inline locales/es_ES/pcsx2_Iconized.po]: "CPUEmotion" debería estar separado. Siendo siglas inglesas, a lo mejor deberíamos sustituir CPU por procesador.
Yo pondría "frecuencia reloj" en lugar de frecuencia, aunque se entiende igual.

File: locales/es_ES/pcsx2_Iconized.po [hunk 9]
```diff
@@ -698,6 +755,11 @@ msgid ""
 "starvation problems, E.g. Shadow of the Colossus. More often than not this "
 "is harmful to performance and causes FPS readouts to be inaccurate."
 msgstr ""
+"Hace que el Emotion Engine salte ciclos, de forma que los programas de VU se "
```
Comment (RafaelTrepaUnCarballo) [inline locales/es_ES/pcsx2_Iconized.po]: "Hace que el Emotion Engine **se** salte ciclos"

**Comment by resadent** [comment]
Thank you @RafaelTrepaUnCarballo, I will improve those lines you mentinoned when I have some time.

**Comment by resadent** [comment]
I've made some changes according to the suggestions, please have a look at them and tell me if you like them. Thanks.

File: locales/es_ES/pcsx2_Main.po [hunk 10]
```diff
@@ -103,93 +96,80 @@ msgstr "Formato de imagen ISO no reconocido"
...
 
 #: pcsx2/MTGS.cpp:877
-msgid ""
-"The MTGS thread has become unresponsive while waiting for the GS plugin to "
-"open."
+msgid "The MTGS thread has become unresponsive while waiting for the GS plugin to open."
 msgstr ""
-"El flujo de MTGS ha dejado de resonder mientras esperaba a que se iniciara "
-"el plugin GS."
+"El flujo de MTGS ha dejado de resonder mientras esperaba a que se iniciara el plugin GS."
```
Comment (RafaelTrepaUnCarballo) [inline locales/es_ES/pcsx2_Main.po]: El **hilo** de MTGS

Comment (RafaelTrepaUnCarballo) [reply locales/es_ES/pcsx2_Main.po]: El **hilo** de MTGS

**Comment by lightningterror** [comment]
Is it ready for merge or still some work left to be done?

**Comment by resadent** [comment]
Seems all good to me.

**Comment by lightningterror** [comment]
@RafaelTrepaUnCarballo you ok with it?

**Comment by RafaelTrepaUnCarballo** [comment]
@lightningterror Yes

**Pull request merged**